plugins/woocommerce/src/Internal/Logging/RemoteLogger.php

```diff
@@ -221,7 +220,7 @@ private function log( $level, $message, $context ) {
 				self::LOG_ENDPOINT,
 				array(
 					'body'     => wp_json_encode( $body ),
-					'timeout'  => 2,
+					'timeout'  => 3,
```

Comment:
Increase the timeout from 2 to 3 seconds to avoid timeouts on slow sites. The request is non-blocking so it shouldn't affect the performance of the site.